README.md

```diff
@@ -180,6 +180,10 @@ The following build task will watch the source code for changes and compile auto
 ```
 yarn run dev
 ```
+If changes aren't detected, you might need to increase `fs.inotify.max_user_watches` by running the command:
+```
+echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```

Comment:
Doesn't work on Arch, so maybe just a link to https://webpack.js.org/configuration/watch/#not-enough-watchers.